src/Mvc/Extensions.ApiDescription.Client/src/build/Microsoft.Extensions.ApiDescription.Client.props

```diff
@@ -1,35 +1,58 @@
 ﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
 <Project>
   <PropertyGroup>
-    <_ApiDescriptionTasksAssemblyTarget
-        Condition="'$(MSBuildRuntimeType)' == 'Core'">netstandard2.0</_ApiDescriptionTasksAssemblyTarget>
-    <_ApiDescriptionTasksAssemblyTarget
-        Condition="'$(MSBuildRuntimeType)' != 'Core'">net461</_ApiDescriptionTasksAssemblyTarget>
-    <_ApiDescriptionTasksAssemblyPath>$(MSBuildThisFileDirectory)/../tasks/$(_ApiDescriptionTasksAssemblyTarget)/Microsoft.Extensions.ApiDescription.Tasks.dll</_ApiDescriptionTasksAssemblyPath>
-    <_ApiDescriptionTasksAssemblyTarget />
+    <_ApiDescriptionClientAssemblyTarget
+        Condition="'$(MSBuildRuntimeType)' == 'Core'">netstandard2.0</_ApiDescriptionClientAssemblyTarget>
+    <_ApiDescriptionClientAssemblyTarget
+        Condition="'$(MSBuildRuntimeType)' != 'Core'">net461</_ApiDescriptionClientAssemblyTarget>
+    <_ApiDescriptionClientAssemblyPath>$(MSBuildThisFileDirectory)/../tasks/$(_ApiDescriptionClientAssemblyTarget)/Microsoft.Extensions.ApiDescription.Client.dll</_ApiDescriptionClientAssemblyPath>
```

Comment:
Is this really relevant anymore? MSBuild supports netstandard2.0 everywhere .NET Core runs.

Reply:
If the assembly has any dependencies that aren't in netstandard _as implemented in .NET 4.7.2_, I'd leave this in. Netstandard task assemblies are probably much more viable now that VS is on 4.7.2, but they've historically been a problem and splitting into core and full implementations has been required.

Reply:
I'm mentioning it because we've been doing ns2.0 in Razor for a while now with no problems. 

Since this is new code it should be fine as long as we test it in VS right?

Reply:
Yeah, as long as it works in VS, command line msbuild.exe, and command line msbuild.exe from a Build Tools install of VS, you should be good to go.

Reply:
I'll file an issue deferring this cleanup and testing in to Preview 7

Reply:
I've added this to #4923 since the extra bits do no harm. This is all an implementation detail and it'll take time to do the necessary testing&hellip;